git_cc/update.py

```diff
@@ -1,11 +1,17 @@
 """Update the git repository with Clearcase manually, ignoring history"""
 
-from common import *
-import sync, reset
+from __future__ import print_function
+
+from .common import *
+from . import reset
+from . import sync
+
 
 def main(message):
```

Comment:
FWIW `update` was always a giant hack, for use in case of emergencies. Are you using `git-cc rebase` at all? Of course it's more complicated and can break for lots of silly reasons, but that's how I normally used it.